tests/acceptance/features/bootstrap/FeatureContext.php

```diff
@@ -706,14 +710,23 @@ public function getIdOfElement(string $user, string $element): int {
 					  </d:prop>
 					</d:propfind>'
 		);
-		$xmlBody = $propfindResponse->getBody()->getContents();
-		$responseXmlObject = new SimpleXMLElement($xmlBody);
+	}
+
+	public function getIdOfElement(string $user, string $element): int {
+		$propfindResponse = $this->getElementResponse($user, $element);
+		$this->theHTTPStatusCodeShouldBe(207, "", $propfindResponse);
```

Comment:
```suggestion
		$this->theHTTPStatusCodeShouldBe(207, "", $propfindResponse);
```
write a readable message for this?